### DISCUSSION TITLE
Make area states trigger scenes - Feature request

### DISCUSSION THREAD

**Discussion by felix63** [jseidl/magic-areas#188] [Ideas / Feature Requests]
Would it be possible to choose different colors and/or brightness level or select scenes? The use-case is that I would like the lights to be dimmed and have a different color depending on night/sleep mode.

**Comment by Bodge-IT** [comment]
Scenes! +1

**Comment by jseidl** [comment]
Brightness and color control are expected to be handled by third-party integrations (MA is very opinionated!). I particularly use https://github.com/basnijholt/adaptive-lighting/ to control levels throughout the day.

Let me know if that is what you looking for

**Comment by Bodge-IT** [comment]
No, being able to select a lighting scene would be optimum choice.  Say for example dark state or accented state.

@jseidl Thanks for all your hard work this week.  Looks like MA is now in much better place!

**Comment by jseidl** [comment]
> No, being able to select a lighting scene would be optimum choice. Say for example dark state or accented state.

Got it, renamed the issue to reflect the actual feature.

> @jseidl Thanks for all your hard work this week. Looks like MA is now in much better place!

Thanks! @caphm also had a good amount of work done on this release with the multiple step config flow and the idea and first implementation of the event-based IPC. I still have some very cool features he did for the light groups that will come in the next minor release. Indeed I agree that MA 3 is WAY BETTER than MA 2. And much more reliable!
